### ISSUE TITLE
Save action still disabled after deleting a tag

### ISSUE THREAD

**Issue by trejkaz** [jaquadro/NBTExplorer#22]
I had opened a file and deleted a tag representing an item in a player's inventory, but the Save button didn't appear after making the change.

I found a workaround where if you pick a simple integer value, edit it to another value and back again, Save will now be enabled and will properly save the file with the tag deleted.

**Comment by jaquadro** [comment]
Did you open an individual file, or were you in a directory view (like when you first open the program?)

**Comment by jaquadro** [comment]
Based on the other issue you opened, I assume this is mac version?

**Comment by trejkaz** [comment]
Mac version, yep. I was in a directory view. Though I was trying to open the player.dat file directly, it wouldn't actually let me select it so I had opened the directory it was inside.

**Comment by hoblin** [comment]
Same issue in mac version

**Comment by glentakahashi** [comment]
Got the same issue as well. Thanks @trejkaz for the fix!!! Saved me a ton of pain

**Comment by JaSpr** [comment]
It's 2.5 years later and I'm still having this issue.

I went File > Open (not "Open Directory") to open a single region's `.mca` file.  Found the item I wanted to delete, deleted it, and the save button was not available. @trejkaz's workaround saved my ass.

**Comment by ScottRadish** [comment]
Yup still a problem. This was the only solution anywhere on the internet. Thanks, @trejkaz.
